10up/simple-podcasting
# Enhancement/41 Podcasts Taxonomy term(s) added in block settings

### Description of the Change

* This PR adds the podcasts taxonomy panel picker in the Block settings.
* Please note, that the taxonomy selection would be still connected with the post itself, not the block.


Closes #41 

### Alternate Designs

![image](https://user-images.githubusercontent.com/25176325/190158164-3ca6c1a3-00e4-4698-b25c-0338575bd6f1.png)

### Possible Drawbacks

* Not a drawback but, the default taxonomy panel under the "Post" tab is still there because if we remove it by making `show_in_rest: true`, we can not replicate it under the settings tab too. If we still need that removed, we may have to add `display: none` to it.

### Verification Process

1. Switch to the PR branch.
2. Run `npm run build`.
3. Create a new/edit existing post.
4. Select the Podcast block.
5. See the podcast taxonomy panel under the block settings.
6. Make sure it works.

### Checklist:






### Changelog Entry

> Added - Podcasts Taxonomy term(s) added in block settings

### Credits

Props @helen @jeffpaul @faisal-alvi
